apps/shared/avifpng.c

```diff
@@ -347,6 +418,15 @@ avifBool avifPNGRead(const char * inputFilename,
         rowPointers[y] = &rgb.pixels[y * rgb.rowBytes];
     }
     png_read_image(png, rowPointers);
+    if (!ignoreICC && hasGama && hasChrm && !hasSrgb && !hasICC) {
+        if (!avifIsSrgbChromaticities(chrm)) {
+            fprintf(stderr,
+                    "WARNING: PNG file contains unsupported chromaticity "
+                    "information (cHRM chunk different from sRGB). Colors may "
```

Comment:
> cHRM chunk different from sRGB

It could be misunderstood as "sRGB chunk", but there is no such chunk here.

Reply:
Reworded.